AspNetCoreGuidance.md

```diff
@@ -199,3 +199,21 @@ app.Use(async (next, context) =>
     context.Response.Headers["test"] = "value";
 });
 ```
+
+✔️**GOOD** This examples uses `Response.OnStarting`. 
```

Comment:
This examples uses `Response.OnStarting` to set the headers before the response headers are flushed to the client.